consensus/safety-rules/src/main.rs

```diff
@@ -27,7 +27,7 @@ fn main() {
         .is_async(config.logger.is_async)
         .level(config.logger.level)
         .init();
-
+    METRICS_PUSHER.start();
```

Comment:
for example, here, we could just instantiate and keep the metrics pusher until exit and drop. Now we won't drop... similarly, we could instantiate one from within the SafetyRulesManager

Reply:
Done